compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -3875,6 +3898,48 @@ private boolean checkReturnValidityInTransaction(AnalyzerData data) {
                 && data.withinTransactionScope;
     }
 
+    private void analyzeInvocationParams(BLangInvocation iExpr, AnalyzerData data) {
+        if (iExpr.symbol == null) {
+            return;
+        }
+
+        BType invocableType = Types.getReferredType(iExpr.symbol.type);
+        BInvokableSymbol invokableSymbol = ((BInvokableSymbol) iExpr.symbol);
+        int parameterCountForPositionalArgs = ((BInvokableType) invocableType).getParameterTypes().size();
+
+        int i = 0;
+        boolean foundNamedArg = false;
+        for (BLangExpression expr : iExpr.argExprs) {
+            switch (expr.getKind()) {
+                case NAMED_ARGS_EXPR:
+                    foundNamedArg = true;
+                    reportIfDeprecatedUsage(((BLangNamedArgsExpression) expr).varSymbol, expr, expr.pos);
+                    i++;
+                    break;
+                case REST_ARGS_EXPR:
+                    if (foundNamedArg) {
+                        continue;
```

Comment:
codecov warnings? Shall we check and update please?

Reply:
We are not coming to this point since we getting errors in the type checker. Do we need to add tests here?

Reply:
Ack. We can't add compiler test then.